### ISSUE TITLE
Web - Step 6 - Topic 2 - Preventing Recurrence - Data Tables Missing TH and scope="row'

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#78]
The first, second, and third TABLE need to have TH and scope="row" added to the first column.

Step 6 - Topic 2 - Preventing Recurrence - https://dolcontentdev.opadev.dol.gov/ilab/complychain/build/steps/6/topic/2

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we address please?

**Comment by jfc3-dol** [comment]
If you have any questions for this issue or any other issues, ask me in the comments.

**Comment by Abychalil** [comment]
Colton Daily added a comment - 16/Jul/19 12:34 AM (#260 in JIRA)

Not sure the ask. Why does a "th" tag need added with scope="row"?

**Comment by jfc3-dol** [comment]
The reason the first column needs to be th scope="row" to a simple TABLE is that it allows the user to know which item a given cell on that row is related too and similar for the column being th scope="col" so they know that information too.

An example would be the TABLE contains a person's name and a few dates in the columns and rows.

Now when a user gets to 12/13/2017, they can use the CTRL + ALT + number pad 5 key to find out the column and row information too. So instead of only hearing "12/13/2017" it would be nice to know what date that is such as start date, end date, last paid date, etc. all those are different and important too. So by using scope="col" and scope="row," the screen reader will read "Start date John Q. Smith 12/13/2017". It's much more useful to the user that only giving them a date.

Here is an article that will explain how to make accessible TABLEs which also talks about adding a CAPTION and a SUMMARY statement, etc.

https://www.456bereastreet.com/archive/200410/bring_on_the_tables/

**Comment by jfc3-dol** [comment]
Thanks for adding the TH scope="col" for the two-column headings, but there is now TH scope="row" code on the first content items of each row for the first three TABLEs. The last TABLE needs to user IDs and HEADERS to get this to work properly because there are six rows that SPAN one at the bottom of the TABLE.

Having both allows the assistive technology user (screen reader, etc.) to know what column heading and row information that a particular cell is related too.

**Comment by KyleLivingstonDOL** [comment]
ZTI, can we please correct and implement for all tables that follow this similar format?

**Comment by Abychalil** [comment]
created new ticket in our Jira for adding TH scope for the third table. We will be looking into it sooner. Thanks!
@KyleLivingstonDOL

**Comment by jfc3-dol** [comment]
@Abychalil and @KyleLivingstonDOL 

The "TH" and scope="row" has not been added to the first column of the TABLE for the content section, so assistive technology users (screen reader, etc.) will be able to tell that column and row information belongs to the other row cells. by using the CTRL + ALT + number pad 5 keys.

**Comment by Abychalil** [comment]
Is there anymore changes we want to make for this ticket? Please advise
@jfc3-dol

**Comment by jfc3-dol** [comment]
@Abychalil 

I don't think there is anything else for these issues besides fixing all the TABLEs on the page.

Here is a link to an article about making TABLEs accessible, and the part I'm talking about is towards the end because he is explaining how to make TABLEs from the start, but it would be a bad thing to read so you can find out about how to do complex TABLEs too.

**Comment by Abychalil** [comment]
We have added the TH scope ='row' and 'col' as shown in the screenshot. Is that the change you are looking forward to?

![TH scope 1](https://user-images.githubusercontent.com/49654920/87093336-086cd380-c20b-11ea-9f21-41ab7d9f045f.PNG)

**Comment by jfc3-dol** [comment]
@Abychalil 

That's what I'm expecting for all the simple TABLEs like these in the application and not only this page.

I can't remember if there is an issue to left-align the TABLE text for ease of reading or not for the TABLEs, but that would be helpful to people with reading issues, especially as you can see for the newly done THs with scope="row" in the first column there are some with two lines of text there is one or a few words in the middle of the wide cell.

You might also want to change the CSS for those items, so the text is not bold too. I will look for the CSS used on DOL.gov for TABLEs like this to fix that issue,

**Comment by jfc3-dol** [comment]
@Abychalil 

I looked at the TABLEs on this page ( https://dolcontentdev.opadev.dol.gov/ilab/complychain/steps/8/topic/3 ) in IE, Chrome, Firefox and they now have a TH with scope="row" for the content section of the TABLE as expected.

So I'm going to close this issue.

**NOTE** - That this fix should apply to other similar simple TABLEs in the application for all languages.
